easybuild/easyconfigs/p/PyTorch-bundle/PyTorch-bundle-2.1.2-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,78 @@
+easyblock = 'PythonBundle'
+
+name = 'PyTorch-bundle'
+version = '2.1.2'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://pytorch.org/'
+description = """PyTorch with compatible versions of official Torch extensions."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+    ('RE2', '2023-08-01'),  # for torchtext
+]
+
+dependencies = [
+    ('CUDA', '12.1.1', '', SYSTEM),
+    ('Python', '3.11.3'),
+    ('PyTorch', version, versionsuffix),
+    ('Pillow-SIMD', '9.5.0'),  # for torchvision
+    ('SentencePiece', '0.2.0'),  # for torchtext
+    ('tqdm', '4.66.1'),  # for torchtext
+    ('double-conversion', '3.3.0'),  # for torchtext
+    ('utf8proc', '2.8.0'),  # for torchtext
+    ('tensorboard', '2.15.1'),  # for torch-tb-profiler
+]
```

Comment:
Can we incorporate the torchaudio into this bundle as well?
It would need SoX and FFmpeg, and maybe a custom build option, otherwise I think it should be straight forward in include from here this PR
https://github.com/easybuilders/easybuild-easyconfigs/pull/19978/files

Reply:
also, you have the ffmpeg patch, but surely this means FFmpeg should also be a dep here already?